### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HotelCalifornia**
### Description of the Change
Update information about installing Okapilib
### Benefits
People should now know how to install Okapilib
### Possible Drawbacks
um
### Verification Process
Seeking advice on specific wording (specifically the bit about "modern" versions of PROS 3)
### Applicable Issues
#8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Documentation is hosted on the PROS website: [OkapiLib Docs](https://pros3.purdu
 
 ## Installing
 
-You can use the PROS CLI to install OkapiLib. In your terminal, inside the project you want to install OkapiLib in, run ????????????
+You can use the PROS CLI to install OkapiLib. In your terminal, inside the project you want to install OkapiLib in, run `pros conduct apply okapilib`. Note that in modern versions of PROS 3, Okapilib will be installed by default in new projects.
```
Comment (Octogonapus) [inline README.md]: Maybe change `modern` to `recent`? I'm not sure. `modern` doesn't sound like the right word.

**Review by Octogonapus — CHANGES_REQUESTED** [review]
Maybe change `modern` to `recent`?

File: README.md (hunk 1 above)
Comment (HotelCalifornia) [reply README.md]: my thoughts exactly

Comment (HotelCalifornia) [reply README.md]: i would specify which versions, but i don't remember which one that got added in

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Documentation is hosted on the PROS website: [OkapiLib Docs](https://pros3.purdu
 
 ## Installing
 
-You can use the PROS CLI to install OkapiLib. In your terminal, inside the project you want to install OkapiLib in, run ????????????
+You can use the PROS CLI to install OkapiLib. In your terminal, inside the project you want to install OkapiLib in, run `pros conduct apply okapilib`. Note that in newer (post-beta) versions of PROS 3, Okapilib will be installed by default in new projects.
```
Comment (HotelCalifornia) [inline README.md:14]: wording can be changed after everyone on beta versions has moved on

**Review by Octogonapus — APPROVED** [review]

**Pull request merged**